README.md

```diff
@@ -1,7 +1,17 @@
-<img src="img/AAA_long_plain.svg" width="75%" alt="AwesomeAndroidAlternatives">
-
-## This is the list of THE BEST apps according to me. You can prove me wrong on the [contributing page](CONTRIBUTING.md).
+<p align="center">
+<img src="img/AAA_long_plain.svg" width="45%" alt="AwesomeAndroidAlternatives"/>
+</p>
+
+## Introduction 
+This is a list of `THE BEST FOSS apps` according to [us](https://github.com/Psyhackological/AAA/graphs/contributors).You can prove us wrong on the [contributing page](CONTRIBUTING.md).
+>
```

Comment:
If I am not mistaken only one `>` is enough and then the rest will be block-quoted too.

Example:
> I am typing
something
to prove
my point.

``` markdown 
Example:
> I am typing
something
to prove
my point.
```